sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala

```diff
@@ -73,6 +73,11 @@ case class CreateFunctionCommand(
       s"is not allowed: '${databaseName.get}'")
   }
 
+  // Redefine a virtual function is not allowed
+  if (FunctionsCommand.virtualOperators.contains(functionName.toLowerCase(Locale.ROOT))) {
+    throw new AnalysisException(s"It's not allowed to redefine virtual function '$functionName'")
```

Comment:
what's the error message if users try to redefine `=`?

Reply:
> what's the error message if users try to redefine `=`?

can't use `create function = ....`, since `=` is a reserved key, we should use 
```
create function `=` ....
```

Error message:
```
org.apache.hadoop.hive.ql.metadata.HiveException: InvalidObjectException(message:= is not a valid object name);
org.apache.hadoop.hive.ql.metadata.HiveException: InvalidObjectException(message:> is not a valid object name);
org.apache.hadoop.hive.ql.metadata.HiveException: InvalidObjectException(message:!= is not a valid object name);

but `case` `between` can be registered.
```

as @HyukjinKwon  methoned, we should fix this ambiguity between functions and operators at the end.

Reply:
If we can't fix the problem completely here, let's keep it unchanged and fix them all together later.

Reply:
> If we can't fix the problem completely here, let's keep it unchanged and fix them all together later.

Ok, have remove these code.